lib/models/different_building_direction_handler.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:concordia_go/models/direction_handler.dart';
+import 'package:concordia_go/models/direction_object.dart';
+import 'package:concordia_go/models/direction_request.dart';
+import 'package:concordia_go/models/node.dart';
+import 'package:concordia_go/services/direction_chain.dart';
+import 'package:concordia_go/services/indoor_path_service.dart';
+
+class DifferentBuildingDirectionHandler implements DirectionHandler {
+  DirectionHandler _nextHandler;
+
+  @override
+  void setNext(DirectionHandler handler) {
+    _nextHandler = handler;
+  }
+
+  @override
+  void handle(DirectionRequest request) {
+    if (canHandle(request)) {
+      final Node exitNode = Node('990000');
```

Comment:
Whats this random 990000? Should put this in a named variable :)

Reply:
From the node naming convention, documented in issue #21  , nodes with a prefix of 99 are exits. So in this case, we are passing the exit, which every building should have, as the destination for indoor-to-outdoor navigation